app/components/windows/Settings.vue.ts

```diff
@@ -42,19 +42,19 @@ export default class SceneTransitions extends Vue {
 
   settingsData = this.settingsService.getSettingsFormData(this.categoryName);
   icons: Dictionary<string> = {
-    General: 'fa fa-th-large',
-    Stream: 'fa fa-globe',
-    Output: 'fa fa-microchip',
-    Video: 'fa fa-film',
-    Audio: 'fa fa-volume-up',
-    Hotkeys: 'fa fa-keyboard-o',
-    Advanced: 'fa fa-cogs',
-    API: 'fa fa-file-code-o',
+    General: 'icon-overview',
+    Stream: 'fas fa-globe',
+    Output: 'fas fa-microchip',
+    Video: 'fas fa-film',
+    Audio: 'icon-audio',
+    Hotkeys: 'icon-settings',
+    Advanced: 'fas fa-cogs',
+    API: 'far fa-file-code',
```

Comment:
what is the difference between `fas` and `far` ?

Reply:
FontAwesome 5 switched to far and fas for 'regular' and 'solid'. For example a circle that is filled in is fas, circle with only stroke is far

Reply:
Oh, good to know